### PR TITLE
Richtext: adding iframes as custom html and add renderer for iframes

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: components/Richtext.js [hunk 1]
```diff
@@ -29,6 +32,8 @@ export const Richtext = ({ text }) => {
   }
 
   const collapsibleRenderer = ({ TDefaultRenderer, tnode,  ...props }) => {
+    const isCollapsible = tnode.domNode.attribs.class === 'collapsible-item'
+
     if (isCollapsible(tnode)){
```
Comment (goapunk) [inline components/Richtext.js:37]: this should now only be `isCollapsible`

Comment (goapunk) [inline components/Richtext.js:37]: this should now be `isCollapsible` instead of a function

Comment (goapunk) [inline components/Richtext.js:37]: this should now just be `isCollapsible` instead of a function

**Review by goapunk — CHANGES_REQUESTED** [review]

File: components/Richtext.js (hunk 1 above)
Comment (khamui) [reply components/Richtext.js]: aiai...okay fixed

**Review by goapunk — APPROVED** [review]
cool

**Pull request merged**